app/src/main/java/com/duckduckgo/app/browser/webview/MaliciousSiteBlockerWebViewIntegration.kt

```diff
@@ -197,9 +242,12 @@ class RealMaliciousSiteBlockerWebViewIntegration @Inject constructor(
         processedUrls.clear()
     }
 
-    override fun onSiteExempted(url: Uri) {
+    override fun onSiteExempted(
```

Comment:
Can we add some test to this method? it seems they are missing. I was wondering about for example: what if url is not a valid URI?

Reply:
I can add some tests, but we don't do any validation. If the Uri is invalid, nothing will break, but it'll never get matched either. All URLs used in this case come from the WebView though, so they won't be invalid, and I don't think we should do validation in every single step of the way, as it would be terrible for performance.